index.js

```diff
@@ -2577,6 +2579,8 @@
   //. Takes a unary predicate and a value of any type, and returns logical
   //. negation of applying predicate to the value.
   //.
+  //. See also (`not`)[#not]
```

Comment:
Wasn't sure if this is a desired reference so left it in a separate commit for review, let me know if this good to be squashed into main commit